client/scss/components/_dropdown-button.scss

```diff
@@ -0,0 +1,136 @@
+// stylelint-disable selector-attribute-name-disallowed-list
+$separator: 1px solid theme('colors.white-15');
+$radius: theme('borderRadius.sm');
+
+.w-dropdown-button {
+  // Cosmetic details based on live tooltip placement,
+  // implemented CSS-only by using Tippy.js `data-placement` and `:has`.
```

Comment:
This is really hard to follow unless you’re atuned with logical corner radius properties. The RTL gif in the README shows that this works.